api/internal.yaml.template

```diff
@@ -25,12 +25,241 @@ paths:
+        - name: signature
+          in: header
+          description: The signature of the HTTP request, signed by the client with its private key.
+          required: true
+          schema:
+            $ref: "#/components/schemas/LollipopSignature"
+      responses:
+        '200':
+          description: "Success response"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/FastLoginResponse"
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/BadRequest"
+        '401':
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Unauthorized"
+        '500':
+          description: Server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/InternalServerError"
 components:
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: X-Functions-Key
   schemas:
+    InternalServerError:
+      description: InternalServerError
+      allOf:
+        - $ref: "#/components/schemas/ProblemDetail"
+        - type: object
+          properties:
+            status:
+              type: integer
+              enum: [500]
+              example: 500
+              description: >-
+                The HTTP status code generated by the origin server for this
+                occurrence of the problem.
+          required:
+            - status
+    BadRequest:
+      description: Bad Request
+      allOf:
+        - $ref: "#/components/schemas/ProblemDetail"
+        - type: object
+          properties:
+            status:
+              type: integer
+              enum: [400]
+              example: 400
+              description: >-
+                The HTTP status code generated by the origin server for this
+                occurrence of the problem.
+          required:
+            - status
+    Unauthorized:
+      description: Bad Request
+      allOf:
+        - $ref: "#/components/schemas/ProblemDetail"
+        - type: object
+          properties:
+            status:
+              type: integer
+              enum: [401]
+              example: 401
+              description: >-
+                The HTTP status code generated by the origin server for this
+                occurrence of the problem.
+          required:
+            - status
     ApplicationInfo:
       type: object
       properties:
         message:
           type: string
       required:
-        - message
\ No newline at end of file
+        - message
+    FastLoginResponse:
+      type: object
+      properties:
+        saml_response:
+          type: string
+          minLength: 1
+      required:
+        - saml_response
+    ProblemDetail:
```

Comment:
We can safely use `ProblemJson`

Reply:
thank you. solved in https://github.com/pagopa/io-functions-fast-login/pull/8/commits/3fd6b7def8fe64e52fb5da73e2043a6612eeb066